spec/02-integration/13-vaults/02-env_vault_spec.lua

```diff
@@ -55,12 +55,11 @@ describe("Environment Variables Vault", function()
     finally(function()
       helpers.unsetenv("TEST_ENV_JSON")
     end)
-    local res, err = get("{vault://env/test_env_json/username}")
+    local res, err = get("{vault://env/test_env_json:username}")
```

Comment:
This sounds like a breaking change to me. We have to offer some sort of migration path.